docs/typst.md

```diff
@@ -0,0 +1,77 @@
+# Writing high-quality Typst files
+
+No specific code style is mandated, but two spaces of indent and kebab-case for
+variable and function names are recommended.
+
+## Use package specification in imports
+
+When writing a package, it is better to use the full package specification
+when importing your own package, instead of relative imports. In other words,
+it is better to write:
+
+```typ
+#import "@preview/my-package:1.0.0": my-function
+```
+
+than:
+
+```typ
+#import "../lib.typ": my-function
+```
+
+This is especially true for example files. The reasoning here is that
```

Comment:
Perhaps, this could be folded into the paragraph above? E.g. "When writing examples for a package, ..."